toolkit/tools/imagegen/configuration/kernelcommandline.go

```diff
@@ -17,11 +17,12 @@ import (
 //   - ExtraCommandLine: Arbitrary parameters which will be appended to the
 //     end of the kernel command line
 type KernelCommandLine struct {
-	CGroup           CGroup      `json:"CGroup"`
-	ImaPolicy        []ImaPolicy `json:"ImaPolicy"`
-	SELinux          SELinux     `json:"SELinux"`
-	EnableFIPS       bool        `json:"EnableFIPS"`
-	ExtraCommandLine string      `json:"ExtraCommandLine"`
+	CGroup             CGroup      `json:"CGroup"`
+	ImaPolicy          []ImaPolicy `json:"ImaPolicy"`
+	SELinux            SELinux     `json:"SELinux"`
+	EnableFIPS         bool        `json:"EnableFIPS"`
+	EnableGrubMkconfig bool        `json:"EnableGrubMkconfig"`
```

Comment:
I'm 50/50 on this being here or a member of the systemconfig parent. We already have `BootType` as part of the config.

Reply:
It just tells the tools where to put the grub binaries. I was just thinking that the new flag could live next to the boottype flag since they both affect the style of our boot configuration. Consider it an optional request/suggestion.